### PR TITLE
5548: fix margin, padding and fontsizes

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to issue

https://platform.dandigbib.org/issues/5548

#### Description

Fix margin between elements
Fix font size 

#### Screenshot of the result

image

### CODE REVIEW

**Review by jeppekroghitk — APPROVED** [review]

File: src/stories/Library/Lists/list-reservation/list-reservation.scss [hunk 1]
```diff
@@ -93,10 +89,15 @@ $list-reservation-space-desktop: 24px;
 
   p {
     margin-top: 2px;
+    font-size: 14px;
+    margin-bottom: 16px;
   }
```
Comment (spaceo) [inline src/stories/Library/Lists/list-reservation/list-reservation.scss:94]: We have talked about using our spacing variables in general. Why not using them here?

**Review by spaceo — CHANGES_REQUESTED** [review]
Looks good. One question though...

File: src/stories/Library/Lists/list-reservation/list-reservation.scss (hunk 1 above)
Comment (sinejespersen) [reply src/stories/Library/Lists/list-reservation/list-reservation.scss]: Changed

**Review by spaceo — APPROVED** [review]
LGTM 👍

**Pull request merged**